src/bees-context.cc

```diff
@@ -825,13 +825,13 @@ BeesContext::set_root_fd(Fd fd)
 	// Start queue producers
 	roots();
 
-	BEESLOG("returning from set_root_fd in " << name_fd(fd));
+	BEESLOGINFO("returning from set_root_fd in " << name_fd(fd));
 }
 
 void
 BeesContext::blacklist_add(const BeesFileId &fid)
 {
-	BEESLOG("Adding " << fid << " to blacklist");
+	BEESLOGWARN("Adding " << fid << " to blacklist");
```

Comment:
That's INFO, maybe even DEBUG.  It's bees hiding its own files from itself.

Reply:
I chose DEBUG.